cryptus-app/components/market_viewer/collection_row/collection_row.module.scss

```diff
@@ -0,0 +1,52 @@
+@use "/styles/variables" as v;
+
+.container {
+  display: flex;
+  flex: 1 1 auto;
+  flex-direction: row;
+  align-items: center;
+  text-align: center;
+
+  width: 100%;
+  height: v.$collection_height;
+  position: sticky;
+}
+
+.infos {
+  display: flex;
+  flex: 1 1 auto;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: flex-start;
+  align-items: center;
+  text-align: center;
+
+  width: 35%;
+  height: v.$collection_height;
+}
+
+.graph {
+    display: flex;
+    flex: 10 1 auto;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: center;
+    align-items: center;
+    text-align: left;
+
+    width: 50%;
```

Comment:
use px (50% would only really work if we wanted to do a 50-50, but then again, px is preferred